pkg/webhook/operatingsystemconfig/testdata/configure-rsyslog.sh

```diff
@@ -13,17 +13,40 @@ function configure_auditd() {
     mv /etc/audit/rules.d /etc/audit/rules.d.original
   fi
 
+  restart_auditd=false
+
   if [[ ! -d /etc/audit/rules.d ]]; then
     mkdir -p /etc/audit/rules.d
   fi
   if ! diff -rq /var/lib/rsyslog-relp-configurator/audit/rules.d /etc/audit/rules.d ; then
     rm -rf /etc/audit/rules.d/*
     cp -L /var/lib/rsyslog-relp-configurator/audit/rules.d/* /etc/audit/rules.d/
-    if [[ -f /etc/audit/plugins.d/syslog.conf ]]; then
-      sed -i 's/no/yes/g' /etc/audit/plugins.d/syslog.conf
-    fi
     augenrules --load
-    systemctl restart auditd
+    restart_auditd=true
+  fi
+
+  if [[ -f /etc/audit/plugins.d/syslog.conf ]] && grep -Fxq "active = no" "/etc/audit/plugins.d/syslog.conf" ; then
+    sed -i 's/no/yes/g' /etc/audit/plugins.d/syslog.conf
+    restart_auditd=true
+  fi
+
+  if ! systemctl is-active --quiet auditd.service ; then
+    # Ensure that the auditd service is running.
+    systemctl start auditd.service
+  elif [ "${restart_auditd}" = true ]; then
```

Comment:
```suggestion
  elif [ "$restart_auditd" = true ]; then
```
Could we drop the {}? I think the expression is simple enough for them to not carry any benefit